package.json

```diff
@@ -18,10 +18,13 @@
   },
   "homepage": "https://github.com/evscott/hackapp-api#readme",
   "dependencies": {
+    "body-parser": "latest",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
+    "nodemon": "^2.0.2",
     "pg": "^7.15.1",
-    "body-parser": "latest"
+    "swagger-jsdoc": "^3.5.0",
```

Comment:
This is super cool. I pulled it down, and it's so convenient how you can test the API directly! That will be really useful.

Reply:
Yeah! That's a tremendous feature. I'm looking forward to _testing_ it.